scripts/squad-run-node.sh

```diff
@@ -91,7 +96,9 @@ run() {
 }
 
 init_env_lite() {
-    yum install -y screen
+    yum install -y -q screen
+    # install tccli
+    pip3 install -q tccli
```

Comment:
add the installation of python3 and pip in front, even if it comes pre-installed, we need to make sure it it there in case we decide to change the image

Reply:
good thanks